### PR TITLE
Add Cabana HDF5 particle output

### PR DESCRIPTION
**Pull request by streeve**
Default to HDF5 output; if not available, use Silo output. If neither are compiled with Cabana, warn that no particle output is available

### CODE REVIEW

**Review by pabloseleson — APPROVED** [review]

**Comment by streeve** [comment]
@pabloseleson can you take another look? This has changed a lot since your approval

File: .github/workflows/CI.yml
Comment (streeve) [inline .github/workflows/CI.yml:44]: This should be updated in the readme

**Comment by streeve** [comment]
Blocked by #27

**Comment by streeve** [comment]
@pabloseleson this should be ready to be tested again

**Review by pabloseleson — APPROVED** [review]

**Pull request merged**
